.circleci/config.yml

```diff
@@ -0,0 +1,24 @@
+version: 2.1
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
+jobs:
+  build:
+    docker:
+      - image: bitriseio/android-ndk:latest
```

Comment:
I think we should use explicit (pinned) version, not the latest and update it manually.
If NDK gets updated in the image then project may stop building.

Reply:
I agree. Pinned version is better here.

Reply:
done